rolling/release-rhel-build.yaml

```diff
@@ -27,6 +27,7 @@ package_blacklist:
 - control_box_rst  # coinor-libipopt-dev has no RPM for RHEL 8
 - gazebo_dev  # Gazebo has no RPM package
 - grbl_ros  # Not yet generated for RHEL
+- ign_rviz_common  # ignition has no RPM packages for RHEL
```

Comment:
Should we also add `ign_rviz` and `ign_rviz_plugins`?

Reply:
They're all downstream of the listed packages.